frontend/language/src/nb.json

```diff
@@ -32,6 +32,7 @@
   "app_data_modelling.landing_dialog_paragraph": "Velkommen til datamodellering! Her kan du laste opp en datamodell du har fra før, eller lage en ny modell du kan bruke i appen din.",
   "app_data_modelling.landing_dialog_upload": "Last opp datamodell",
   "app_data_modelling.upload_xsd": "Last opp",
+  "app_data_modelling.upload_xsd_invalid_name_error": "Filnavnet er ugyldig. Du kan bruke tall og store og små bokstaver fra det norske alfabetet, og understrek, punktum og bindestrek. Navnet må dessuten starte på en bokstav.",
```

Comment:
I wonder if we need `og` before `store`  and `understrek`? because we mentions many things(that should be seperated by commmas) and we use `og `before the last thing? 🤷‍♂️

like this maybe:
Filnavnet er ugyldig. Du kan bruke tall, store og små bokstaver fra det norske alfabetet, understrek, punktum og bindestrek. Navnet må dessuten starte på en bokstav.

Reply:
Tror dette er en tekst @Ildest har jobbet med. Jeg har egt bare gjenbrukt den som er benyttet når man oppretter den ny modell og dermed kan skrive inn et modellnavn. 
Jeg tror ordlyden er som den er fordi `understrek, punktum og bindestrek` er én type gruppering av regler mens hvilke type _bokstaver_ man kan bruke er en annen gruppering. Derfor er det delt opp med `og` to ganger. 
Men jeg reagerte på det selv så kanskje @Ildest har mulighet til å se på en annen formulering? ☺️

Reply:
> Tror dette er en tekst @Ildest har jobbet med. Jeg har egt bare gjenbrukt den som er benyttet når man oppretter den ny modell og dermed kan skrive inn et modellnavn. Jeg tror ordlyden er som den er fordi `understrek, punktum og bindestrek` er én type gruppering av regler mens hvilke type _bokstaver_ man kan bruke er en annen gruppering. Derfor er det delt opp med `og` to ganger. Men jeg reagerte på det selv så kanskje @Ildest har mulighet til å se på en annen formulering? ☺️

Det er riktig tenkt @standeren, at det er på grunn av at det er to grupper tegn i teksten. Det kan høres litt rart ut med to og-er, men jeg prøvde å unngå ordet "samt" som er gammeldags for og. Jeg tror den må stå slik den er. Den siste setningen vil jeg ha endret til "Navnet må alltid starte med en bokstav"